en/peripherals/remote_id.md

```diff
@@ -49,6 +53,10 @@ The TX and RX on the flight controller must be connected to the RX and TX on the
 | 3 (blk) | RX (IN)  |      |
 | 4 (blk) | GND      | 0    |
 
+Note that Cube ID does not support firmware updates in PX4 (only Ardupilot).
```

Comment:
I think you mean that you have to use ArduPilot/Mission Planner to update cube id firmware. I will replace this with a link to the Cube ID docs.